packages/react-utilities/src/hooks/useOnClickOutside.ts

```diff
@@ -62,8 +68,8 @@ export const useOnClickOutside = (options: UseOnClickOrScrollOutsideOptions) =>
     };
 
     if (!disabled) {
-      element?.addEventListener('click', conditionalHandler);
-      element?.addEventListener('touchstart', conditionalHandler);
+      element?.addEventListener('click', conditionalHandler, !!capture);
+      element?.addEventListener('touchstart', conditionalHandler, !!capture);
```

Comment:
```suggestion
      element?.addEventListener('click', conditionalHandler, true);
      element?.addEventListener('touchstart', conditionalHandler, true);
```

As we are targeting the specific hook I suggest to make `capture` a default thing there 😏 In all cases that we had in Northstar these event handlers should run in `capture` mode to be evaluated before React handlers:

https://github.com/microsoft/fluentui/blob/8de04ed56820cfdb7c13d9c108b0eae596cd3215/packages/fluentui/react-northstar/src/components/Popup/Popup.tsx#L434

I also see the same in React Spectrum:

https://github.com/adobe/react-spectrum/blob/4b75cbe84157bcd735f7097a41f397d2c5c58543/packages/%40react-aria/interactions/src/useInteractOutside.ts#L66-L67

Reply:
yeah, that does make things simpler